### ISSUE TITLE
Task: null on timings

### ISSUE THREAD

**Issue by JamieSinn** [PaperMC/Paper#1177]
https://i.imgur.com/IYPI1vG.png
https://timings.aikar.co/?id=74f10d712d2143309609553edb27d4a0

Shouldn't happen, so what's up?

**Comment by aikar** [comment]
appears its using a "Local" class, didn't know anyone actually did that...

MinecraftTimings.java:
```java
        final String taskname;
        if (taskClass.isAnonymousClass()) {
            taskname = taskClass.getName();
        } else {
            taskname = taskClass.getCanonicalName();
        }
```

**Comment by aikar** [comment]
while in there, we really should cache task names to an IdentityHashMap too.

**Comment by JamieSinn** [comment]
We used local classes previously with the old timings/stacktraces to allow it to show the line numbers instead of a weird reference.

**Comment by aikar** [comment]
fixed in latest builds for both 1.12 and 1.13

**Comment by JamieSinn** [comment]
you're awesome aikar!
